includes/class-omise-capabilities.php

```diff
@@ -68,6 +72,30 @@ public static function retrieve($pKey = null, $sKey = null)
 		return self::$instance;
 	}
 
+	/**
+	 * @param boolean $isCheckout
+	 * @param boolean $isAdmin
+	 * @param boolean $isThankYouPage
+     * @return boolean
+     */
+    public static function shouldCallApi() {
+		$omiseSettingPages = [ 'omise' ];
+
+		$currentAdminPage = isset( $_GET[ 'page' ] ) ? $_GET[ 'page' ] : '';
+
+		// checkout page but not thank you page
+		// By default thank you page is also part of checkout pages
+		// and we do not need to call capabilities on thank you page.
+		// If endpoint url is `order-received`, it mean thank you page.
+		$isPaymentPage = is_checkout() && !is_wc_endpoint_url( 'order-received' );
+
+		// if from admin panel and is omise page.
```

Comment:
What does it mean? If order was created from admin dashboard?

Reply:
Omise setting page in admin dashboard.